scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3193,7 +3205,7 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     def failMix(advice: Option[String]) = {
       val message = "these statements can't be mixed together"
       val addendum = advice.map(", " + _).getOrElse("")
-      syntaxError(message + addendum, at = parserTokens.head)
+      syntaxError(message + addendum, at = scannerTokens(0))
```

Comment:
I don't think they are equivalent. I believe the equivalence in the new scheme is `new SimpleTokenIterator().next()`